IonDotnet/Timestamp.cs

```diff
@@ -361,7 +361,14 @@ private static int GetOffsetMinutes(string s, int startIdx)
                 throw new FormatException(s);
             }
 
-            return hour * 60 + minute;
+            int offset = hour * 60 + minute;
+
+            if (hour > 23 || minute > 59)
```

Comment:
This doesn't seem to account for negatives. Where is the sign handled?

Would it make more sense for this condition to be `if (offset < MIN_OFFSET || offset > MAX_OFFSET)`, where `MAX_OFFSET` is previously defined as `23 * 60 + 59` and `MIN_OFFSET` is defined as `-MAX_OFFSET`?

Reply:
The signs (-/+) are parsed right before this method is called ([here](https://github.com/amzn/ion-dotnet/blob/master/IonDotnet/Timestamp.cs#L257)) and here just the value is calculated.

The reason for not using ``MAX_OFFSET`` is that technically ``"2007-01-01T00:00-10:70"`` is within the accepted offset range, but not a valid one.